frontend/src/index.pug

```diff
@@ -69,15 +69,15 @@ html
           #tabs-wrapper(ref="tabWrapper", slot="right")
             .tab-content.panel-padding
               .tab-panes
-                v-authorship#tab-authorship.tab-pane(
```

Comment:
Agrees to remove `.tab-pane`, but we can rename .`tab-panes` to `.tab-pane` also.